src/utils/formValidationUtils.js

```diff
@@ -0,0 +1,16 @@
+export const validateNumberWithinRange = ({
+  min,
+  max,
+  value,
+  errorReturn = false,
+}) => {
+  if (!value) return
```

Comment:
This is going to cause problems, what happens when `value = 0`?

Reply:
I included this line so that it can return properly when the value is undefined for non-required fields. So that it will not return any errors for those that are not required. When there's a value such as `value = 0`, it will proceed to the range logic either `min or max`. I'm not sure if there any other problems that could happen